app.rb

```diff
@@ -0,0 +1,89 @@
+require './person'
+require './student'
+require './teacher'
+require './book'
+require './rental'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_all_books
+    puts 'All books'
+    @books.each { |book| puts book }
+  end
+
+  def list_all_people
+    puts 'All people'
+    @people.each { |person| puts person }
+  end
+
+  def create_person(name, type)
+    if type == 'teacher'
+      person = Teacher.new(name)
+    elsif type == 'student'
+      person = Student.new(name)
```

Comment:
- [ ] I got this error while trying to create a student. Kindly check for bugs and fix this issue. Hints of the error can be seen in this screenshot 

![image](https://github.com/eudondian/opp-school-library/assets/63473584/3624f601-f633-4394-9c03-a6757a1ada84)